lib/src/component_client.dart

```diff
@@ -25,4 +25,11 @@ import 'package:w_flux/src/store.dart';
 /// This FluxComponent, intended for use on the client, utilizes the
 /// [BatchedRedraws] mixin to throttle redraws down to one per animation frame.
 abstract class FluxComponent<ActionsT, StoresT>
-    extends FluxComponentCommon<ActionsT, StoresT> with BatchedRedraws {}
+    extends FluxComponentCommon<ActionsT, StoresT> with BatchedRedraws {
+  @override
+  void componentWillUnmount() {
+    // ensure that unmounted components don't batch render
+    shouldBatchRedraw = false;
```

Comment:
@maxwellpeterson-wf That's correct.

I was always under this assumption, since it wasn't explicitly stated in the docs, but here's a source that confirms that instances aren't reused: https://github.com/facebook/react/issues/4770#issuecomment-136928760